src/lib/rest/rest.cpp

```diff
@@ -1138,14 +1144,22 @@ static int connectionTreat
 */
 static int restStart(IpVersion ipVersion, const char* httpsKey = NULL, const char* httpsCertificate = NULL)
 {
-  bool   mhdStartError = true;
-  size_t memoryLimit   = connectionMemory * 1024; // connectionMemory are kilobytes
+  bool      mhdStartError  = true;
+  size_t    memoryLimit    = connMemory * 1024; // connMemory is expressed in kilobytes
+  MHD_FLAG  flag           = MHD_USE_THREAD_PER_CONNECTION;
```

Comment:
`flag` is rather generic as variable name... what about `connectionType` or `serverMode` or similar?